sdcm/cluster.py

```diff
@@ -4419,20 +4419,14 @@ def __init__(self, params):
     def gemini_version(self):
         if not self._gemini_version:
             try:
-                result = self.nodes[0].remoter.run('cd $HOME; ./gemini --version', ignore_status=True)
+                result = LocalCmdRunner().run(f'docker run --rm {self.params.get("stress_image.gemini")} '
```

Comment:
Wouldn't it be better to run it on loader - so image wouldn't be downloaded on runner but on loader and later reused?

Reply:
changed it to the loader